extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -499,6 +539,8 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
    */
   @SourceDsl
   class Builder(val sourceId: String) {
+
+    internal var rawData: Any? = null
```

Comment:
Can we save a `CompletableFuture<String>` instead of the rawData here, we can start processing the data already when user set it, and we can check if the future is done and set the geojson immediately or if it's not done, we could append async setting data property in the `completableFuture#thenAccept` callback.

Reply:
As discussed - not viable because `CompletableFuture` requires higher min API and using just `Future` is not sufficient.